pkg/enqueue-bundle/DependencyInjection/EnqueueExtension.php

```diff
@@ -171,6 +171,18 @@ private function registerJobQueueDoctrineEntityMapping(ContainerBuilder $contain
             return;
         }
 
+        $config = $container->getExtensionConfig('enqueue');
+
+        if (!empty($config)) {
+            $processedConfig = $this->processConfiguration(new Configuration(false), $config);
+
+            foreach ($processedConfig as $name => $modules) {
+                if (isset($modules['job']) && false === $modules['job']['default_mapping']) {
```

Comment:
I think we can avoid `for` loop here. 

Something like this should do:

```php
isset($processedConfig['enqueue']['job']['default_mapping']) && $processedConfig['enqueue']['job']['default_mapping'] == false
```

Reply:
Hm, `$processedConfig` does not have `enqueue` key. By default it would have a `default` key, but I guess it could be any string defined by the user, no? In the documentation it is defined as `key`:

<img width="737" alt="Screenshot 2020-11-24 at 10 38 25" src="https://user-images.githubusercontent.com/31573482/100069734-bdfa9500-2e41-11eb-8690-a2e19b77feb3.png">

Reply:
well, then it's ok.